draft-ietf-mls-protocol.md

```diff
@@ -1178,18 +1097,17 @@ A group member generates this message using the following steps:
   direct path between the new leaf and the new root
 
 The public key of the ephemeral key pair is placed in the
-`ephemeral` field of the GroupAdd message.  The computed direct path
+`ephemeral` field of the Add message.  The computed direct path
 is placed in the `add_path` field.
```

Comment:
nit: should we include info about  identity frontier and ratcher_frontier inclusion in the above para ?

Reply:
I agree that those definitions would make sense.

Reply:
Done, though note that this whole section will get overwritten by #59  :)